src/frontend/helpers/index.ts

```diff
@@ -11,49 +11,45 @@ import { GogInstallInfo } from 'common/types/gog'
 
 import { install, launch, repair, updateGame } from './library'
 import fileSize from 'filesize'
-const { ipcRenderer } = window.require('electron')
-const readFile = async (file: string) => ipcRenderer.invoke('readConfig', file)
+const readFile = window.api.readConfig
 
-const writeConfig = async (
-  data: [appName: string, x: unknown]
-): Promise<void> => ipcRenderer.invoke('writeConfig', data)
+const writeConfig = window.api.writeConfig
 
 const notify = ([title, message]: [title: string, message: string]): void =>
-  ipcRenderer.send('Notify', [title, message])
+  window.api.notify([title, message])
```

Comment:
Hm, I think that would hurt the clarity a bit, you'd then think it's just a regular function while it's actually calling the backend. `window.api` isn't too much to write

Reply:
Yeah I agree with @CommandMC. I think developers coming from other electron projects might find it a bit unclear as to whether it's a function or backend call. It also might add a bunch of boilerplate with
`const {loginPage, handleKofi...} = window.api`
and even after that, there's nothing stopping the dev from writing 
`window.api.loginPage(...args)`
which might be confusing as there are now two equivalent ways of calling the api function.

Reply:
I agree that `window.api` is ok to write. Although `window` at least in my head would never be associated with backend :P
But ok, let's keep as it is.
It is good to decide this kind of stuff on refactors since that will be the pattern we will use from now on.